aziotctl/aziotctl-common/src/restart.rs

```diff
@@ -0,0 +1,59 @@
+use std::io::{self, Write};
+use std::process::Command;
+
+use crate::ServiceDefinition;
+
+pub fn restart(services: &[&ServiceDefinition]) {
+    // stop all services
+    for ServiceDefinition {
+        service,
+        sockets: _,
+    } in services
```

Comment:
This multi-line destructure inside a for loop looks a little... weird?

It might be a bit more verbose, but something like this seems easier to grok at a glance:

```rust
for service in services.iter().map(|s| s.service)
```

You used this in a whole bunch of places, so if you decide to change it, make sure to fix the other ones as well.

Reply:
its really just the auto-formatter that makes it look weird. I agree its weird split on multiple lines. single line it reads fine. Since we always use the auto-formatter, I'll switch to map.

```
for ServiceDefinition {service, sockets: _,} in services {

}
```